gdeep/trainer/trainer.py

```diff
@@ -132,6 +145,8 @@ def forward(self, x):
     registered_hook: Optional[
         Callable[[int, Optimizer, ModelExtractor, Optional[SummaryWriter]], Any]
     ] = None
+    regularizer: "Regularizer"
+    lamda: float
```

Comment:
I think this lambda has to have a default of, say, `1.`

Reply:
Done. In the end this lamda was moved to be a property of a regularizer, not trainer. This makes the regularizer parameters more similar (you pass them all to the regularizer). Also, as it is not absolutely required for all regularizers, this avoids the tricks previously present in defining e.g. elastic net that uses several lambdas